lfucg-exactions/client/js/components/AccountForm.js

```diff
@@ -278,12 +277,18 @@ function mapDispatchToProps(dispatch, params) {
         onSubmit(event) {
             event.preventDefault();
             if (selectedAccount) {
-                dispatch(putAccount(selectedAccount));
-                hashHistory.push(`account/summary/${selectedAccount}`);
+                dispatch(putAccount(selectedAccount))
+                .then((data) => {
```

Comment:
Put functions do not always return results, and .then() functions after a Put break in Internet Explorer